spring-cloud-dataflow-platform-kubernetes/src/main/java/org/springframework/cloud/dataflow/server/config/kubernetes/KubernetesTaskPlatformFactory.java

```diff
@@ -61,7 +56,9 @@ public Launcher createLauncher(String account) {
 		KubernetesTaskLauncher kubernetesTaskLauncher = new KubernetesTaskLauncher(
 				kubernetesProperties, kubernetesClient, containerFactory);
 
-		Scheduler scheduler = getScheduler(schedulerProperties, kubernetesClient);
+		KubernetesSchedulerProperties kubernetesSchedulerProperties = new KubernetesSchedulerProperties();
+		BeanUtils.copyProperties(kubernetesProperties, kubernetesSchedulerProperties);
```

Comment:
It looks like not all the properties are copied into the scheduler properties. I just tried the `limits` and `requests` but they don't seem to be merged into scheduler properties.